odk1-src/form-audit-log.rst

```diff
@@ -90,6 +90,17 @@ You can enable change tracking so that old answers and new answers will be added
 
   audit, audit, track-changes=true
 
+Enumerator identification
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your form needs a record of the identity of the enumerator you can use :tc:`identify-user=true`.
```

Comment:
I think it would be great to add some context as to how this might be useful and to contrast it with `username` in https://docs.opendatakit.org/collect-settings#form-metadata-settings and accessed by https://docs.opendatakit.org/form-question-types/#metadata.

Maybe something like this:

In some data collection workflows, devices are passed between different enumerators for data verification or completion. If you need to know who had the device when data was entered or changed, you can use :tc:`identify-user=true`.

If each device will be used by a single enumerator, you can use the server username or [settings username](https://docs.opendatakit.org/collect-settings#form-metadata-settings) and the [`username` metadata question](https://docs.opendatakit.org/form-question-types/#metadata). This will write the username to each submission instead of to the audit log.

Reply:
Will have a look at this

Reply:
Chose to do this in a "tip". Felt to me like it was nice to have the core text be fairly functional and then have the tip provide information around the other possible feature.